_includes/header.liquid

```diff
@@ -4,7 +4,8 @@
   <section class="page-header">
     <div class="wrapper">
       <div class="page-header__logo">
-        <a href="/" class="logo">{{ site.title }}</a> <span class="badge--beta">beta</span>
+        <a href="/" class="logo">{{ site.title }}
```

Comment:
I personally make it easy for me to catch those things by indenting all levels unless you only have a string you wrap in a tag.

```html
<a href="/" class="logo">
	{{ site.title }}
	<span class="uikit--version">{% uikit_version %}</span>
</a>
```

But no need to change this... :)